composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "netatmo/netatmo-api",
+    "type": "library",
+    "description": "Netatmo API client implementation - PHP SDK",
+    "keywords" : ["netatmo", "api", "sdk"],
+    "homepage" : "https://dev.netatmo.com/",
+    "license": "MIT",
+    "minimum-stability": "stable",
```

Comment:
you can omit this property as this package does not have any dependency and the `minimum-stability` is only read from root packages (i.e packages that will require `netatmo/netatmo-api`)